public/scripts/utils.js

```diff
@@ -0,0 +1,32 @@
+function getCookie(name) {
```

Comment:
Common utils functions that are used in both stats and main ejs files (currently however, the main.ejs file is not using this, as these functions are present in the file inline)